core/pluginz/ezpz.toml

```diff
@@ -1,3 +1,4 @@
 [ezpz_pluginz]
```

Comment:
lets give user option to use either dedicated ezpz.toml or add [tool.ezpz] to their pyproject.toml